### ISSUE TITLE
Version 2.8 not working in Firefox

### ISSUE THREAD

**Issue by midros** [garoloup/SendToQNAP#6]
Hi all

Firefox version 24.0.2 (64-bit)

Add-on version 2.8 does not connect to download station.
If I install version 2.7 everything is fine.

**Comment by garoloup** [comment]
Hi midros,

I removed this v2.8 and will check.
Thanks for your feedback.

**Comment by midros** [comment]
I am glad I helped.
Keep up the good work.
This add-on is very helpful to me.

**Comment by lukjod** [comment]
Yes had to go back to 2.7, newest did not connect to Qnap DS no local nor remote
